src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -4028,14 +4028,31 @@ public object VisitCommand(CommandAst commandAst)
                     var splatTest = element;
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for splatting.Splatting members and indexed data with the using statement.Error occurred in Context block
```powershell
Cannot process argument transformation on parameter 'Test'. Cannot convert the "using: properly" value of type "System.String" to type "System.Management.Automation.ScriptBlock".
At D:\a\1\s\test\powershell\Language\Scripting\Splatting.Tests.ps1:148 char:44
```